crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -922,6 +924,87 @@ end
 # DATA LOADERS
 # =================================================================================================
 
+#! Loads account data from the advice inputs into the _current_ account's memory section.
+#!
+#! Inputs:
+#!   Operand stack: [account_id_prefix, account_id_suffix]
+#!   Advice map: {
+#!     ACCOUNT_ID: [[account_id_suffix, account_id_prefix, 0, account_nonce],
+#!                          VAULT_ROOT, STORAGE_COMMITMENT, CODE_COMMITMENT],
+#!     STORAGE_COMMITMENT: [[STORAGE_SLOT_DATA]],
+#!     CODE_COMMITMENT: [num_procs, [ACCOUNT_PROCEDURE_DATA]]
```

Comment:
Is this correct? I think for account code the entry is `CODE_COMMITMENT |-> [ACCOUNT_PROCEDURE_DATA]`. (i.e., there is no `num_procs` there).